### DISCUSSION TITLE
AuroraAssetEditor requiring .NET Framework 4.8 but Windows 7 or 10 is not able to install it...

### DISCUSSION THREAD

**Discussion by Elusiv3** [XboxUnity/AuroraAssetEditor#16] [Q&A] [answered]
Hi guys, 

Trying to get [Aurora Asset Editor v1.4.2](https://github.com/XboxUnity/AuroraAssetEditor/releases/tag/v1.4.2) working but Microsoft.Net Framework is giving me the following installation certificate due to error:

"0x80096005 - The timestamp signature and/or certificate could not be verified or is malformed."

I found the following youtube video on how to fix it but didn't work for me:

https://www.youtube.com/watch?v=tFAk5S0Gpls

Running Windows 7 btw.

When I tried on Windows 10 I got the following:  "Blocking Issues:  The .NET Framework 4.8 is not supported on this operating system."

Can anyone please help?

**Comment by Swizzy** [comment]
What version of Windows 10 are you running when testing it?

Make sure you run Windows Update - you might even get the .NET 4.8 from there instead if you do - most likely, you're running a outdated version that isn't supported anymore, but running the latest version should be good - you want to do that anyways for security reasons.

**Reply by Elusiv3** [reply]
ahhhh... ok, that makes sense.  

No money for new hardware, all my machines are older macs running Bootcamp/OpenCore Legacy Patcher.

I'll report back once I figure out a solution...

Thank you for your time and all that you do!!  You are legedary!!

**Reply by Elusiv3** [reply, marked as answer]
woo hoo, got it working!  What a rollercoaster ride getting bootcamp up and running the most up to date Windows 10 on my legacy macbookpro 5.2.  Thanks again to you and your team Swizzy!!
